client/src/QuestionsAndAnswerComponents/question.jsx

```diff
@@ -5,17 +5,17 @@ import { Card, Container, Row, Col } from 'react-bootstrap';
 class Question extends Component {
   constructor(props) {
     super(props);
+
     this.state = {
       questions: [],
-      productId: this.props.productId,
     };
     this.fetchQuestionList = this.fetchQuestionList.bind(this);
     this.componentDidMount = this.componentDidMount.bind(this);
   }
 
-  fetchQuestionList(prodId) {
+  fetchQuestionList() {
 
-    fetch(`http://3.21.164.220/qa/questions/?product_id=${prodId}&count=4`)
+    fetch(`http://3.21.164.220/qa/questions/?product_id=${this.props.productId.id}&count=4`)
```

Comment:
Should **${this.props.productId.id}** be **${this.props.productId}** ??

Reply:
I'm not sure if this is the bug! If its not, I can see whats going on when I pull down and run everything!